<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormAddRule</class>
 <widget class="QWidget" name="FormAddRule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="2">
    <widget class="QComboBox" name="comboBoxCmd">
     <item>
      <property name="text">
       <string>All</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>unlinkat</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>openat</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>close</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>read</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>exec</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>connect</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>socket</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>module</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>sendto</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>recvfrom</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>mount</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>bind</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLineEdit" name="lineEditPath"/>
   </item>
   <item row="8" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>69</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="2">
    <widget class="QPushButton" name="pushButtonOK">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>logtime</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QComboBox" name="comboBoxRe">
     <item>
      <property name="text">
       <string>All</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Success</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Fail</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>result</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2" colspan="2">
    <widget class="QDateTimeEdit" name="dateTimeEditRemind"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Audittype</string>
     </property>
    </widget>
   </item>
   <item row="8" column="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>69</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>file_path</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>All time</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="lineEditUsername"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
